cli/crawl-cli.js

```diff
@@ -117,14 +137,40 @@ async function run(inputUrls, outputPath, verbose, logPath, numberOfCrawlers, da
 
     /**
      * @param {URL} url
-     * @param {object} data
+     * @param {{testStarted: number, testFinished: number, data: {screenshot: string}}} data
      */
     const dataCallback = (url, data) => {
         successes++;
         updateProgress(url.toString());
 
+        crawlTimes.push([data.testStarted, data.testFinished, data.testFinished - data.testStarted]);
+
         const outputFile = createOutputPath(url);
         fs.writeFileSync(outputFile, JSON.stringify(data, null, 2));
+
+        // temp name for the screenshot is scored in data. rename the screenshot to match the file crawl file
+        if (data.data.screenshot) {
+            const screenshotFilename = `${outputPath}/screenshots/${url.hostname}_${outputFile.match(/_([a-z0-9]{4})\.json/)[1]}.jpg`;
+            fs.move(data.data.screenshot, screenshotFilename, err => {
```

Comment:
Could we just use `fs.rename` and avoid the need to use the `fs-extra` module entirely?

Reply:
rename doesn't work across filesystems